src/components/SMBEnumeration/SMBEnumeration.tsx

```diff
@@ -162,55 +151,56 @@ const SMBEnumeration = () => {
             setPid(result.pid);
             setOutput(result.output);
         } catch (e: any) {
-            setOutput(e);
+            setOutput(e.message || "An error occurred during the scan.");
         }
     };
 
-    // Clears the output by setting it to an empty string.
+    // Clears the output state
     const clearOutput = useCallback(() => {
         setOutput("");
         setAllowSave(false);
         setHasSaved(false);
-    }, [setOutput]);
+    }, []);
 
+    // Updates the component's state after the user has successfully saved the scan results
     const handleSaveComplete = useCallback(() => {
         setHasSaved(true);
         setAllowSave(false);
     }, []);
 
+    // Rendering the form and related components
     return (
         <form
             onSubmit={form.onSubmit((values) =>
-                onSubmit({ ...values, speed: selectedSpeedOption, scripts: selectedScriptOption })
+                onSubmit({
+                    ...values,
+                    speed: selectedSpeedOption,
+                    script: selectedScriptOption,
+                })
             )}
         >
             {LoadingOverlayAndCancelButton(loading, pid)}
             <Stack>
-                {UserGuide(title, description_userguide)}
+                {UserGuide(title, descriptionUserGuide)}
                 <TextInput label={"Target IP or Hostname"} required {...form.getInputProps("ip")} />
                 <TextInput label={"Port"} required {...form.getInputProps("port")} placeholder={"Example: 445"} />
-
                 <NativeSelect
                     label={"Scan Speed"}
                     value={selectedSpeedOption}
                     onChange={(e) => setSelectedSpeedOption(e.target.value)}
                     title={"Scan speed"}
                     data={speeds}
-                    placeholder={"Select a scan speed. Default set to T3"}
                     description={"Speed of the scan, refer: https://nmap.org/book/performance-timing-templates.html"}
                 />
-
                 <NativeSelect
                     label={"SMB Script"}
                     value={selectedScriptOption}
                     onChange={(e) => setSelectedScriptOption(e.target.value)}
                     title={"SMB Enumeration Script"}
                     data={scripts}
                     required
-                    placeholder={"Select an SMB Enumeration Script to run against the target"}
                     description={"NSE Scripts, refer: https://nmap.org/nsedoc/scripts/"}
                 />
-
                 <Button type={"submit"}>Scan</Button>
```

Comment:
The format of the button should be Start <name of tool>
```suggestion
                <Button type={"submit"}>Start {title}</Button>
```